kraken/litmus/common_litmus.py

```diff
@@ -0,0 +1,98 @@
+import kraken.invoke.command as runcommand
+import yaml
+import logging
+import time
+import sys
+
+
+# Install litmus and wait until pod is running
+def install_litmus(version):
+    runcommand.invoke("kubectl apply -f "
+                      "https://litmuschaos.github.io/litmus/litmus-operator-%s.yaml" % version)
```

Comment:
you might want to consider disabling the [telemetry](https://docs.litmuschaos.io/docs/faq-general/#does-litmus-track-any-usage-metrics-on-the-test-clusters) that is being collected for the operator:
`  oc patch -n litmus deployment.apps/chaos-operator-ce --type=json --patch '
   [
     {
       "op": "add",
       "path": "/spec/template/spec/containers/0/env/-",
       "value": {
         "name": "ANALYTICS",
         "value": "FALSE"
       }
     }
   ]'`

Reply:
Added after the installation of litmus